src/components/Wizard/CreateVmWizard/providers/VMwareImportProvider/VMWareImportProvider.js

```diff
@@ -107,7 +111,8 @@ export class VMWareImportProvider extends React.Component {
   });
 
   componentDidUpdate() {
-    const { vmwareToKubevirtOsConfigMap, v2vvmware } = this.props;
+    const v2vvmware = flatten(this.props, 'v2vvmware', {});
+    const vmwareToKubevirtOsConfigMap = flatten(this.props, 'vmwareToKubevirtOsConfigMap', undefined);
```

Comment:
do we really have to specify undefined value here?

Reply:
no, leftover